configuration/analyzer_config.json

```diff
@@ -1878,6 +1878,31 @@
     "secrets": {},
     "params": {}
   },
+  "Phishstats": {
+    "type": "observable",
+    "python_module": "phishstats.PhishStats",
+    "description": "check if the observable is a phishing site",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": true,
+    "observable_supported": [
+      "ip"
+    ],
+    "config": {
+      "soft_time_limit": 100,
+      "queue": "long"
+    },
+    "secrets": {
+      "api_key_name": {
+        "env_var_key": "PHISTATS_API_KEY",
+        "type": "string",
+        "default": null,
+        "description": "Optional API key for Phishstats.",
+        "required": false
+      }
```

Comment:
phishstats API does not require an API key so this is not needed.

Reply:
Noted